src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,15 @@
+.cards {
+  display: flex;
+  justify-content: space-between;
+  padding: 50px 40px;
+  flex-wrap: wrap;
+  margin: 0 auto;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.